### PR TITLE
Jetpack: update type scale on Backup / Scan

### PR DESCRIPTION
**Pull request by keoshi**
#### Changes proposed in this Pull Request

* Update type scale on Jetpack Backup/Scan.
* Use typeface variable on Jetpack Backup/Scan upsell pages.

cc @sixhours so that you're aware of this change.

#### Screenshots

Before | After
--|--
image | image
image | image
image | image
image | image
image | image

#### Testing instructions

**For Jetpack.com**

* Spin up this PR by running `yarn run start-jetpack-cloud`.
* Open `http://jetpack.cloud.localhost:3000/` and select any Jetpack site with Scan and Backup.
* Confirm you see the changes and that everything looks good.

**For WordPress.com, Jetpack sites**

* Spin up this PR by running `yarn start`.
* Open `http://calypso.localhost:3000/` and select any Jetpack site with Scan and Backup.
* Confirm you see the changes and that everything looks good.

**For WordPress.com, Simple sites**

* Spin up this PR by running `yarn start`.
* Open `http://calypso.localhost:3000/` and select any Simple site with a Free, Personal or Premium plan.
* Go to `Jetpack > Backup`.
* Confirm you see the changes and that everything looks good.

### CODE REVIEW

**Comment by folletto** [comment]
Wonderful alignment work! 🌟

File: client/my-sites/backup/style.scss [hunk 1]
```diff
@@ -67,8 +67,8 @@
 .backup__subheader {
 	margin: 20px 16px 8px;
 
-	font-family: 'Recoleta', Georgia, 'Times New Roman', Times, serif;
-	font-size: rem( 21px );
+	font-family: $brand-serif;
```
Comment (sixhours) [inline client/my-sites/backup/style.scss:70]: If you `@extend .wp-brand-font` here, you'll take advantage of the language selectors, which means folks who don't have full support for Recoleta won't see "ransom note" style characters.

See:

https://github.com/Automattic/wp-calypso/blob/00bb0f424a8816da7f11e3f7a37a3a065156e683/packages/typography/sass/fonts.scss#L17-L26

File: client/my-sites/scan/style.scss [hunk 2]
```diff
@@ -72,8 +72,8 @@
 .scan__subheader {
 	margin: 20px 16px 8px;
 
-	font-family: 'Recoleta', Georgia, 'Times New Roman', Times, serif;
-	font-size: rem( 21px );
+	font-family: $brand-serif;
```
Comment (sixhours) [inline client/my-sites/scan/style.scss:75]: Same here with `.wp-brand-font`

File: client/my-sites/backup/style.scss (hunk 1 above)
Comment (keoshi) [reply client/my-sites/backup/style.scss:70]: Amazing tip, wasn't aware of this piece of code! Thanks, @sixhours.

Comment (keoshi) [reply client/my-sites/backup/style.scss]: Addressed in https://github.com/Automattic/wp-calypso/pull/43685/commits/a31d53a17873fa2dc0e84acdb387e9e00c9ae747

Comment (simison) [reply client/my-sites/backup/style.scss]: Does that work by extending tho? 🤔 

I'd imagine you have to add `wp-brand-font` class to the component?

Comment (keoshi) [reply client/my-sites/backup/style.scss]: @simison FWIW, it seems to work: 

image

Comment (simison) [reply client/my-sites/backup/style.scss]: Excellent! 🎉

**Pull request merged**